packages/backend/README.md

```diff
@@ -109,3 +109,20 @@ Example                                            | Description
 ### More
 
 See the `scripts` defined in [`package.json`](./package.json).
+
+
+### Minimal setup for local development
+If you work locally, you have to prepare this minimal database setup for the backend to work correctly.
+
+#### First create the database:
+CREATE DATABASE modos_dev;
+
+#### Then install the extensions:
+CREATE EXTENSION IF NOT EXISTS adminpack;
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE EXTENSION IF NOT EXISTS fuzzystrmatch;
+CREATE EXTENSION IF NOT EXISTS tablefunc;
+CREATE EXTENSION IF NOT EXISTS postgis;
+CREATE EXTENSION IF NOT EXISTS postgis_topology;
+CREATE EXTENSION IF NOT EXISTS pgrouting;
+CREATE EXTENSION IF NOT EXISTS citext;
```

Comment:
I think it would be better to have a chapter "prerequisite", where you just say that a Postgres DB instance with postgis and pgrouting is needed (with the version number indcated).

Reply:
Last commit should fix this.